### PR TITLE
generate recaptcha enterprise library

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**

### CODE REVIEW

File: google-cloud-recaptcha_enterprise/.rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+AllCops:
+  Exclude:
+    - "google-cloud-recaptcha_enterprise.gemspec"
+    - "lib/google/**/*"
+    - "Rakefile"
+    - "test/**/*"
+    - "acceptance/**/*"
```
Comment (quartzmo) [inline google-cloud-recaptcha_enterprise/.rubocop.yml]: Add `TargetRubyVersion: 2.2` here?

File: google-cloud-recaptcha_enterprise/Gemfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+source "https://rubygems.org"
+
+gemspec
+
+# WORKAROUND: builds are having problems since the release of 3.0.0
+# pin to the last known good version
+gem "public_suffix", "~> 2.0"
```
Comment (quartzmo) [inline google-cloud-recaptcha_enterprise/Gemfile]: Is this pin still required?

Comment (TheRoyalTnetennba) [reply google-cloud-recaptcha_enterprise/Gemfile]: I don't know. It's set by the generator. Running tests for this lib with 3.0.0 didn't fail, but I don't know what the original issue was.

Comment (TheRoyalTnetennba) [reply google-cloud-recaptcha_enterprise/Gemfile]: I can't get any test failures w/ 3.0.0 on any of the libs with this comment on it.

File: google-cloud-recaptcha_enterprise/.rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+AllCops:
+  TargetRubyVersion: 2.2
+  TargetRubyVersion: 2.2
```
Comment (quartzmo) [inline google-cloud-recaptcha_enterprise/.rubocop.yml]: Shoot, did we somehow end up with two lines here setting the target version?

File: google-cloud-recaptcha_enterprise/Gemfile [hunk 4]
```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+
+gemspec
+
+gem "rake", "~> 11.0"
```
Comment (quartzmo) [inline google-cloud-recaptcha_enterprise/Gemfile]: Can this be `gem "rake", "~> 12.0"` as in the #2130 updates?

**Pull request merged**